src/pages/workspace/WorkspaceProfilePage.tsx

```diff
@@ -128,7 +128,7 @@ function WorkspaceProfilePage({policy, currencyList = {}, route}: WorkSpaceProfi
                                 type={CONST.ICON_TYPE_WORKSPACE}
                                 fallbackIcon={Expensicons.FallbackWorkspaceAvatar}
                                 style={[
-                                    isSmallScreenWidth ? styles.mb1 : styles.mb3,
+                                    policy?.errorFields?.avatar ?? isSmallScreenWidth ? styles.mb1 : styles.mb3,
```

Comment:
Should we only be  evalutating `isSmallScreenWidth ? styles.mb1 : styles.mb3` if there's an error? Before we always hit this check